docs/learn-staking.md

```diff
@@ -26,7 +26,7 @@ A validator node is required to be responsive 24/7, perform its expected duties
 
 Any potential validators can indicate their intention to be a validator candidate. Their candidacies are made public to all nominators, and a nominator in turn submits a list of any number of candidates that it supports. In the next epoch (lasting several hours), a certain number of validators having the most DOT backing get elected and become active. There are no particular requirements for a DOT holder to become a nominator, though we expect each nominator to carefully track the performance and reputation of validators.
 
-Once the nomination period ends, the NPoS election mechanism takes the nominators and their associated votes as input, and outputs a set of validators of the required size, that maximizes the stake backing of any validator, and that makes the stakes backing validators as evenly distributed as possible. The objectives of this election mechanism are to maximize the security of the network, and achieve fair representation of the nominators. If you want to know more about how NPoS works (e.g. election, running time complexity, etc.), please read [here](http://research.web3.foundation/en/latest/polkadot/NPoS/).
+Once the nomination period ends, the NPoS election mechanism takes the nominators and their associated votes as input, and outputs a set of validators of the required size, that maximizes the stake backing of any validator, and that makes the stakes backing validators as evenly distributed as possible. The objectives of this election mechanism are to maximize the security of the network, and achieve fair representation of the nominators. If you want to know more about how NPoS works (e.g. election, running time complexity, etc.), please read [here](http://research.web3.foundation/en/latest/polkadot/NPoS.html).
```

Comment:
Do you think it makes sense to link to the Phragmen page here?

Reply:
I thought about it but decided against it, as it's not referencing just that part of the election specifically. Also, I think this document will need a further revamp soon, or a separate "basic" version built, because this is possibly the most essential doc we have for new ecosystem participants and it's complex beyond readability for most people not already familiar with most mechanics here. I think we should have learn-staking-basics that dumbs this down a lot and removes a lot of stuff like this paragraph.